hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -356,6 +357,63 @@ def hpat_pandas_series_append_impl(self, to_append):
     return hpat_pandas_series_append_impl
 
 
+@overload_method(SeriesType, 'min')
+def hpat_pandas_series_min(self, axis=None, skipna=True, level=None, numeric_only=None):
```

Comment:
Missed **kwargs

Reply:
we don't need kwargs in this place

Reply:
In this case do we still need to describe kwargs in docstring? I would say not.